addons/binding/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/handler/UniFiControllerThingHandler.java

```diff
@@ -174,6 +174,10 @@ public void handleCommand(ChannelUID channelUID, Command command) {
         logger.debug("Ignoring command = {} for channel = {} - the UniFi binding is read-only!", command, channelUID);
     }
 
+    public @Nullable UniFiController getController() {
+        return controller;
+    }
+
```

Comment:
This is starting to get messy with the inter-dependencies of `UniFiClientThingHandler` depending on both `UniFiControllerThingHandler` and `UniFiController`.

I think I have a better way and sent @scurb a PM on the community to discuss potential collaboration on this PR. It will also help in the future when we want to add more "write" functions to the binding.